framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q

```diff
@@ -1 +1 @@
-select ps_supplycost from ps_5level where dir0=2000 and dir1='q1' and dir2='jan' and dir3=1 and dir4=1.0 or dir4=2.0 order by ps_partkey limit 10;
+select ps_supplycost from ps_5level where dir0=2000 and dir1='q1' and dir2='jan' and dir3=1 and dir4=1.0 or dir4=2.0 order by ps_partkey limit 8;
```

Comment:
I did not understand this change. Can you please explain why limit 10 was changed to limit 8?

Reply:
The table has 8 rows per partkey. When sorted by partkey, the first 8 rows have partkey value 1 but the next two could be any of the 2 rows having partkey value 2. 
The ones being returned by GA branch are different from apache master.

Reply:
Can't we order by ps_supplycost as well?

Reply:
Made the changes as suggested

Reply:
Thanks!